dominion/base/action.py

```diff
@@ -18,6 +18,7 @@ class Action(KingdomCard):
     def play(cls, deck: Deck) -> None:
         super(cls, Card).play(deck)
         deck.discard(cls)
+        deck.actions_remaining -= 1
```

Comment:
I refuse to accept this naming style because it reflects confusing language that no dominion player i believe has ever used consistently.